ulib/experimental/Steel.Reference.fsti

```diff
@@ -151,31 +151,55 @@ val cas_pt (#t:eqtype)
 (** Second version of references: The memory contents are available inside the selector, instead of as an index of the predicate **)
 
 /// An abstract separation logic predicate stating that reference [r] is valid in memory.
-/// (AF) TODO: This should also be indexed by a permission
-val ptr (#a:Type0) (r:ref a) : slprop u#1
+
+val ptrp (#a:Type0) (r:ref a) (p: perm) : slprop u#1
```

Comment:
I think the permission should have the smt_fallback attribute, as for the non-selector version